client/src/pages/choosePlayers.tsx

```diff
@@ -76,57 +87,24 @@ export default function ChoosePlayers() {
     <div>
       <div>Choose Players</div>
       <h2>Names</h2>
-      {names
-        .filter((name) => !name.selected)
-        .map((name) => (
-          <Card
-            style={{ border: 'solid blue', margin: '.25em', width: '50em' }}
-            key={name.name}
-            onClick={() => handleClickName(name)}
-          >
-            <CardContent>
-              <Typography>{name.name}</Typography>
-            </CardContent>
-          </Card>
-        ))}
-      <div>
-        <p>Selected Names</p>
-        {names
-          .filter((name) => name.selected)
-          .map((name) => (
-            <Card
-              style={{ border: 'solid blue', margin: '.25em', width: '50em' }}
-              key={name.name}
-              onClick={() => handleClickName(name)}
-            >
-              <CardContent>
-                <Typography>{name.name}</Typography>
-              </CardContent>
-            </Card>
-          ))}
-      </div>
+      {musicians.map((musician: Name) => (
+        <Card
+          className={`${classes.card} ${musician.selected && classes.selected}`}
+          key={musician.name}
+          onClick={() => handleClickMusician(musician)}
+        >
+          <CardContent>
+            <Typography>{musician.name}</Typography>
+          </CardContent>
+        </Card>
+      ))}
       <div>
         <h2>Instruments</h2>
-        {instruments
-          .filter((instrument) => !instrument.selected)
-          .map((instrument) => (
-            <Card
-              style={{ border: 'solid red', margin: '.25em', width: '50em' }}
-              key={instrument.instrument}
-              onClick={() => handleClickInstrument(instrument)}
-            >
-              <CardContent>
-                <Typography>{instrument.instrument}</Typography>
-              </CardContent>
-            </Card>
-          ))}
-      </div>
-      <Typography>Selected Instruments</Typography>
-      {instruments
-        .filter((instrument) => instrument.selected)
-        .map((instrument) => (
+        {instruments.map((instrument) => (
           <Card
-            style={{ border: 'solid red', margin: '.25em', width: '50em' }}
+            className={`${classes.card} ${
+              instrument.selected && classes.selected
+            }`}
             key={instrument.instrument}
```

Comment:
would it make sense for the property on instrument to be `instrumentName`? That way when we access it we would use `instrument.instrumentName`? or even just `name` might work.... what do you think?